examples/lambdarank/README.md

```diff
@@ -3,15 +3,15 @@ LambdaRank Example
 
 Here is an example for LightGBM to run lambdarank task.
 
-***You should copy executable file to this folder first.***
+***You must follow the [installation instructions](https://lightgbm.readthedocs.io/en/latest/Installation-Guide.html) for the following commands to work. The `lightgbm` binary must be build and available at the root of this project.***
 
 Training
 --------
 
 Run the following command in this folder:
 
-```
-"./lightgbm" config=train.conf
+```bash
+../../lightgbm config=train.conf
```

Comment:
`"` were used here to make this command cross-platform, i.e. for successful run on Windows. Please put them back.